scripts/postcss-filter.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+// post-process CSS using PostCSS
+// (https://github.com/postcss/postcss)
+//
+// This adds vendor prefixes using autoprefixer
+// https://github.com/postcss/autoprefixer
+
+require('es6-promise').polyfill();
+
+var autoprefixer = require('autoprefixer');
+var postcss = require('postcss');
+
+var inputCss = '';
+
+process.stdin.on('data', function (chunk) {
+  inputCss += chunk;
+});
+
+process.stdin.on('end', function () {
+  postcss([autoprefixer])
```

Comment:
Could we be explicit about what browsers we're supporting, perhaps with a `browserslist` config file?